apps/theming-designer/src/components/ThemeDesignerColorPicker.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import { TextField } from '../../../../packages/office-ui-fabric-react/lib/TextField';
+import * as stylesImport from './app.scss';
+import { Stack } from 'office-ui-fabric-react/lib/Stack';
+import { Text } from '../../../../packages/office-ui-fabric-react/lib/Text';
+
+const styles: any = stylesImport;
+
+export const ThemeDesignerColorPicker = (props: any) => {
+  const handleChange = (event: any, newVal: string | undefined) => {
```

Comment:
Consider adding types for `any` throughout... becomes more difficult as the code grows.